### PR TITLE
Document system integration

### PR DESCRIPTION
**Pull request by defagos**
# Description

This PR adds documentation for AirPlay and Control Center integration.

# Changes made

- Self-explanatory.
- Remove playback button transition animation.

# Checklist

- [x] APIs have been properly documented (if relevant).
- [x] The documentation has been updated (if relevant).
- [x] New unit tests have been written (if relevant).
- [x] The demo has been updated (if relevant).
- [x] The playground has been updated (if relevant).

### CODE REVIEW

File: Sources/Player/Player.docc/ControlCenter.md [hunk 1]
```diff
@@ -2,14 +2,31 @@
...
 
 ## Overview
 
-- For AirPlay support please ensure that your application audio session and background capabilities are configured appropriately.
-- Call `becomeActive()` on a player instance to enable AirPlay and Control Center support for it.
-- Call `resignActive()` on a player to disable AirPlay and Control Center support for it (if currently active).
+> Important: Control Center integration is only meaningful for iOS apps.
 
-You need to call `becomeActive()` when transitioning to an immersive player experience for which these integrations make sense. Calling `resignActive()` is most of the time superfluous, except when the same player instance is reused between an immersive playback user interface and a limited playback experience.
+``Player`` natively integrates with the iOS Control Center. Though most of this integration happens automatically your app is still responsible of activating the player instance which must be controlled by the Control Center. It must also provide the metadata (title, artwork image) associated with the ``PlayerItem`` currently being played.
```
Comment (waliid) [inline Sources/Player/Player.docc/ControlCenter.md:14]:
```suggestion
``Player/Player`` natively integrates with the iOS Control Center. Though most of this integration happens automatically your app is still responsible of activating the player instance which must be controlled by the Control Center. It must also provide the metadata (title, artwork image) associated with the ``PlayerItem`` currently being played.
```

File: Sources/Player/Player.docc/ControlCenter.md [hunk 2]
```diff
@@ -2,14 +2,31 @@
...
+
+### Provide player item metadata
+
+The main responsibility of a ``PlayerItem`` loaded into a ``Player`` is to deliver an ``Asset`` to be actually played. Assets do not only convey the URL to be played but can also be attached arbitrary metadata whose only requirement is to conform to the ``AssetMetadata`` protocol.
```
Comment (waliid) [inline Sources/Player/Player.docc/ControlCenter.md:18]:
```suggestion
The main responsibility of a ``PlayerItem`` loaded into a ``Player/Player`` is to deliver an ``Asset`` to be actually played. Assets do not only convey the URL to be played but can also be attached arbitrary metadata whose only requirement is to conform to the ``AssetMetadata`` protocol.
```

File: Sources/Player/Player.docc/ControlCenter.md [hunk 3]
```diff
@@ -2,14 +2,31 @@
...
+
+To associate Control Center metadata with a player item:
+
+1. Create a type which represents your asset metadata and have it conform to ``AssetMetadata``.
+2. Implement the ``AssetMetadata/nowPlayingMetadata()`` method and return the ``NowPlayingMetadata`` which must be displayed in the Control Center when the item is currently being played.
+3. Implement a custom ``PlayerItem`` with a metadata publisher retrieving all metadata required before delivering an asset. Alternatively, and provided you have all metadata and the URL to be played readily available, you can simply use one of the available ``PlayerItem`` construction helpers, supplying the asset metadata to be used at creation time.
+
+### Make a player instance active
+
+Several ``Player`` instances can coexist in an app but only one at most is able to integrate with the Control Center.
```
Comment (waliid) [inline Sources/Player/Player.docc/ControlCenter.md:28]:
```suggestion
Several ``Player/Player`` instances can coexist in an app but only one at most is able to integrate with the Control Center.
```

File: Sources/Player/Player.docc/Player.md [hunk 4]
```diff
@@ -12,11 +12,11 @@ The Player framework provides a complete toolbox to add advanced audiovisual med
 
 Play content easily with a ``Player`` and display its content in the standard AVKit user interface with ``SystemVideoView``, or build an entirely custom user interface starting from a simple ``VideoView``. Play any kind of content from any source by implementing your own ``PlayerItem``s and ``Asset``s, no matter where your content comes from or how it must be played. Track playback with your own ``PlayerItemTracker``, whether for analytics purposes or Quality of Service (QoS) data collection.
```
Comment (waliid) [inline Sources/Player/Player.docc/Player.md:13]:
```suggestion
Play content easily with a ``Player/Player`` and display its content in the standard AVKit user interface with ``SystemVideoView``, or build an entirely custom user interface starting from a simple ``VideoView``. Play any kind of content from any source by implementing your own ``PlayerItem``s and ``Asset``s, no matter where your content comes from or how it must be played. Track playback with your own ``PlayerItemTracker``, whether for analytics purposes or Quality of Service (QoS) data collection.
```

**Review by waliid — APPROVED** [review]

File: Sources/Player/Player.docc/ControlCenter.md (hunk 1 above)
Comment (defagos) [reply Sources/Player/Player.docc/ControlCenter.md:14]: See comment in #662.

File: Sources/Player/Player.docc/Player.md (hunk 4 above)
Comment (defagos) [reply Sources/Player/Player.docc/Player.md:13]: See comment in #662.

File: Sources/Player/Player.docc/ControlCenter.md (hunk 2 above)
Comment (defagos) [reply Sources/Player/Player.docc/ControlCenter.md:18]: See comment in #662.

File: Sources/Player/Player.docc/ControlCenter.md (hunk 3 above)
Comment (defagos) [reply Sources/Player/Player.docc/ControlCenter.md:28]: See comment in #662.

**Review by waliid — APPROVED** [review]

**Pull request merged**